website/src/categories/components/toggle-group/usage.md

```diff
@@ -96,10 +96,8 @@ If the user selections need to persist across different interactions or sessions
 
 |Item|Name| Description | Optional|
 |-|-|-|-|
-|1|Toogle Button |Description |Yes|
-|2|Container |Description |Yes|
-|3|Icons |Description |No|
-|3|Label |Description |No|
+|1|Toogle Button |These are the control button, for more information check the **toggle group**. |Yes|
```

Comment:
```suggestion
|1|Toggle Button |These are the control buttons, for more information check the **toggle group**. |Yes|
```
Should this not say "toggle button" at the end? And then link to the toggle button page?